graphs/rdf-construct.fbp

```diff
@@ -0,0 +1,32 @@
+# rdf-construct.fbp
+
+# Retrieves the source-graph (IRI) from the sparql-endpoint and returns parsed JSON-LD
```

Comment:
So this (in essence) serializes a graph to JSON-LD and then parses it?

Do we need this?  If so, where?